src/phases/replication/startup/StartupPhasesTemplate.yml

```diff
@@ -49,40 +49,40 @@ Document: &Doc
   padding: {^FastRandomString: {length: {^Parameter: {Name: "approxDocumentSize", Default: 100}}}}
 
 # The default indexes that will be used in the 'InsertDataTemplate'.
-IndexesForLoader: &IndexesForLoader  # 10 indexes
-- keys: {t: 1, w: 1}
-- keys: {y: 1}
-- keys: {t: 1, stringShort: 1}
-- keys: {string0: 1}
-- keys: {stringLong: 1}
-- keys: {stringShort: 1, y: 1}
-- keys: {int0: 1, int1: 1, int2: 1}
-- keys: {int3: 1}
-- keys: {int4: 1}
-- keys: {int5: 1, int6: 1}
+IndexesForLoader: &IndexesForLoader
+  - keys: {t: 1, w: 1}
+  - keys: {y: 1}
+  - keys: {t: 1, stringShort: 1}
+  - keys: {string0: 1}
+  - keys: {stringLong: 1}
+  - keys: {stringShort: 1, y: 1}
+  - keys: {int0: 1, int1: 1, int2: 1}
+  - keys: {int3: 1}
+  - keys: {int4: 1}
+  - keys: {int5: 1, int6: 1}
 
 # The default indexes that will be used in the 'CreateIndexesTemplate'.
-IndexesForCreateCmd: &IndexesForCreateCmd  # 10 indexes
-- key: {t: 1, w: 1}
-  name: "index_t_w"
-- key: {y: 1}
-  name: "index_y"
-- key: {t: 1, stringShort: 1}
-  name: "index_t_stringShort"
-- key: {string0: 1}
-  name: "index_string0"
-- key: {stringLong: 1}
-  name: "index_stringLong"
-- key: {stringShort: 1, y: 1}
-  name: "index_stringShort_y"
-- key: {int0: 1, int1: 1, int2: 1}
-  name: "index_int0_int1_int2"
-- key: {int3: 1}
-  name: "index_int3"
-- key: {int4: 1}
-  name: "index_int4"
-- key: {int5: 1, int6: 1}
-  name: "index_int5_int6"
+IndexesForCreateCmd: &IndexesForCreateCmd # 10 indexes
+  - key: {t: 1, w: 1}
+    name: "index_t_w" # 10 indexes
```

Comment:
Did this comment move manually or automatically ?

Reply:
automatically